markdown/1.x/en/documentation/configuration.md

```diff
@@ -73,4 +73,4 @@ document.addEventListener("ffReady", () => { // "ffReady"-event ensures global f
 
 To prevent FACT-Finder Web Components from missing a factfinder response, before they are loaded, dispatching of results to subscribers is cached and deferred until all components are ready to receive the response. As soon as FACT-Finder Web Components have finished loading `ff-communication` calls `factfinder.communication.ResultDispatcher.startDispatching()`. If `ff-communication` is not used, `factfinder.communication.ResultDispatcher.startDispatching()` has to be called manually in JavaScript.
 
-Also we discourage the direct use of custom JavaScript configuration and strongly recommend to configure everything through FACT-Finder Web Components, you might take a closer look at [ff-core.d.ts](https://github.com/FACT-Finder-Web-Components/ff-web-components/blob/master/dist/ff-core.d.ts). Every configuration can also be done using JavaScript.
\ No newline at end of file
+Although entire configuration can be done using JavaScript, we discourage direct use of custom JavaScript to do that and strongly recommend to configure everything through FACT-Finder Web Components. You might take a closer look at [ff-core.d.ts](https://github.com/FACT-Finder-Web-Components/ff-web-components/blob/master/dist/ff-core.d.ts).
```

Comment:
It might come across a little smoother like this:
```suggestion
Although the entire configuration can be done using JavaScript, we discourage direct use of custom JavaScript to do so and strongly recommend to configure everything through FACT-Finder Web Components. You might want to take a closer look at [ff-core.d.ts](https://github.com/FACT-Finder-Web-Components/ff-web-components/blob/master/dist/ff-core.d.ts).
```